BSP/STM32F413/Makefile

```diff
@@ -9,7 +9,7 @@ DEBUG = 1
 # building variables
 ######################################
 # optimization
-OPT = -Og
+OPT = -O3
 
```

Comment:
I'll make a change so it runs with -Og if DEBUG is 1 and -O3 otherwise

Reply:
On second thought, I think a comment would be better because I don't think changing the `DEBUG` variable is significantly easier than changing the `OPT` variable